### PR TITLE
Update docs - Quick Start and User Guide

### PR DESCRIPTION
**Pull request by yokawasa**
### Description

- Add description on Azure blob storage 
- Add description on Google Cloud Storage
- Update quick start guide
  - Add Configure Cloud Storage Provider for storing Gatling reports
  - Add Configure Notification Service Provider and notify Gatling load testing results via Slack 

Relevant issue #
https://github.com/st-tech/gatling-operator/issues/22

### CODE REVIEW

**Comment by yokawasa** [comment]
@akitok  @ksudate  @st-tech/sre-dept 
It's doc update. 
Please check the PR and approve if you don't find anything wired. Please 🙏

**Review by ksudate — APPROVED** [review]
lgtm!!

File: docs/quickstart-guide.md [hunk 1]
```diff
@@ -137,4 +151,131 @@ Simulation MyBasicSimulation completed in 14 seconds
...
+```
+
+Here is a sample Gatling HTML report:
+
+![](../assets/gatling-html-report.png)
+
+Finally, cleanup the Gatling CR manually
+
+```bash
+kubeclt delete gatling gatling-sample01
```
Comment (akitok) [inline docs/quickstart-guide.md:224]:
```suggestion
kubectl delete gatling gatling-sample01
```

File: docs/quickstart-guide.md [hunk 2]
```diff
@@ -137,4 +151,131 @@ Simulation MyBasicSimulation completed in 14 seconds
...
+```yaml
+apiVersion: gatling-operator.tech.zozo.com/v1alpha1
+kind: Gatling
+metadata:
+  name: gatling-sample01
+spec:
+  generateReport: true                     # The flag of generating Gatling report
+  generateLocalReport: false               # The flag of generating Gatling report for each pod
+  notifyReport: false                      # The flag of notifying Gatling report
+  cleanupAfterJobDone: true                # flag of cleanup Gatling resources after the job done
```
Comment (akitok) [inline docs/quickstart-guide.md:171]: nits
```suggestion
  cleanupAfterJobDone: true                # The flag of cleanup Gatling resources after the job done
```

File: docs/quickstart-guide.md [hunk 3]
```diff
@@ -137,4 +151,131 @@ Simulation MyBasicSimulation completed in 14 seconds
...
+```yaml
+apiVersion: gatling-operator.tech.zozo.com/v1alpha1
+kind: Gatling
+metadata:
+  name: gatling-sample01
+spec:
+  generateReport: true                     # The flag of generating Gatling report
+  generateLocalReport: false               # The flag of generating Gatling report for each pod
+  notifyReport: true                       # The flag of notifying Gatling report
+  cleanupAfterJobDone: true                # flag of cleanup Gatling resources after the job done
```
Comment (akitok) [inline docs/quickstart-guide.md:243]: nits
```suggestion
  cleanupAfterJobDone: true                # The flag of cleanup Gatling resources after the job done
```

**Review by akitok — COMMENTED** [review]
@yokawasa 
I made some nitpick comments.
Thanks for the good docs!

**Pull request merged**